src/client/Microsoft.Identity.Client/Internal/Requests/SilentRequest.cs

```diff
@@ -73,6 +74,9 @@ private async Task<IAccount> GetAccountFromParamsOrLoginHintAsync(AcquireTokenSi
 
         internal async override Task PreRunAsync()
         {
+            if (AuthenticationRequestParameters.IsBrokerEnabled)
```

Comment:
What about iOS ? I thought iOS broker does not have a silent flow, so if you have the broker enabled and perform `AcquireTokenSilent` from iOS, this will now fail...

Reply:
resolved